backend/src/main/java/com/woowacourse/momo/group/service/dto/request/GroupRequestAssembler.java

```diff
@@ -9,14 +9,15 @@
 import com.woowacourse.momo.group.domain.duration.Duration;
 import com.woowacourse.momo.group.domain.group.Group;
 import com.woowacourse.momo.group.domain.schedule.Schedule;
+import com.woowacourse.momo.member.domain.Member;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class GroupRequestAssembler {
 
-    public static Group group(GroupRequest request) {
+    public static Group group(Member host, GroupRequest request) {
```

Comment:
Member 객체가 통으로 필요한가요?

Reply:
```suggestion
    public static Group group(Long hostId, GroupRequest request) {
```
위와 같이 선언하는 것보다는, Member 객체를 받아 객체의 id값을 사용하는 것이 보다 안전하다고 판단했습니다!!
`group()` 메서드를 사용하는 측에서 member의 id가 아니라 다른 도메인을 나타내는 id값을 넘길 수도 있으니까요!!